### ISSUE TITLE
Don't add user as a member of its primary group

### ISSUE THREAD

**Issue by deric** [deric/puppet-accounts#69]
User shouldn't be added as member of its primary group, as mentioned in #66
```
test:x:1011:100:Test Account,,,:/home/test:/bin/bash
test:x:100:test
```

**Comment by deric** [comment]
cc: @5obol Problem should be fixed in the master branch, the fix should be part of the next release.

**Comment by deric** [comment]
Released as [v1.5.3](https://github.com/deric/puppet-accounts/releases/tag/v1.5.3).
